### ISSUE TITLE
Missing DrainEvent metadata for SpotITN

### ISSUE THREAD

**Issue by gavinbunney** [aws/aws-node-termination-handler#104]
The spot instance termination notice endpoint is only populating the timestamp of the event in the `DrainEvent` struct.

You can see this in the logs:
```
Got drain event from channel {EventID: Kind:SPOT_ITN Description:Spot ITN received. will be at 2020-03-02T18:20:55Z
```

The documentation for the notice API also only indicates a `time` is returned: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-interruptions.html#instance-action-metadata

Can you confirm that this is working as expected? If so, will probably need to add another step to fetch more instance metadata during these notices, as at the moment the logs (and webhook notifications) are pretty sparse.

Thanks!

Another example of a webhook notice showing the `DrainEvent` fields not being populated:

![image](https://user-images.githubusercontent.com/409207/75709980-9393f680-5c78-11ea-867b-933de58cba64.png)

**Comment by mattrandallbecker** [comment]
Thanks for reporting the issue, we'll investigate the problem. The integration tests we wrote only check if a message was successfully sent, looks like that was a mistake and some of the other data is missing.

Additionally, I agree that we need a richer set of metadata information available to the webhook, we already have the following issue for that : https://github.com/aws/aws-node-termination-handler/issues/94

**Comment by bwagner5** [comment]
fixed and released in v1.3.0
